edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -291,6 +292,7 @@ const LectureViewer: React.FC<{
             horizontal
             style={{
                 flex: 1,
+                backgroundColor: colorScheme == "dark" ? "#181825" : "#e6e9ef",
```

Comment:
That's a fine touch, this way we have a more agreeably background for the PDF when we are viewing the PDF. Nice touch, passing the `colorScheme` hook is great.

Reply:
Yes indeed